### ISSUE TITLE
Request: support for texture filtering

### ISSUE THREAD

**Issue by MastaG** [dethrace-labs/dethrace#277]
This is more like a feature request.
I've been running Dethrace on Linux and it works very well.
However would it be possible to implement texture filtering so it looks like the 3Dfx glide version?
(E.g. smooth textures instead of minecraft-pixelated)

**Comment by dethrace-labs** [comment]
Thanks - yes, its something I think we will add, maybe with a `--3dfx` flag to switch into that mode.

**Comment by MastaG** [comment]
Well it's been more than a year now.
And I see the project getting some traction again.
I believe the difference between 3dfx glide and software rendering is mostly the texture filtering right?
So is adding a -3dfx flag which activates texture filtering a lot of work to implement?

**Comment by dethrace-labs** [comment]
yes, it is a decent amount of work. Carmageddon in 3dfx mode follows a different path to rendering things with transparency, so we will be implementing that code path
